vars/prChanges.groovy

```diff
@@ -1,13 +1,14 @@
 import groovy.transform.Field
 
-public static @Field PR_CHANGES_CACHE = null
+public static @Field PR_CHANGES_CACHE = []
```

Comment:
This is a change that's here only to accommodate the unit test framework. There was a really hard to figure out scope problem related to how the testing framework works.